src/sinks/aws_s_s/sns/integration_tests.rs

```diff
@@ -0,0 +1,197 @@
+#![cfg(all(test, feature = "aws-sns-integration-tests"))]
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏This should more appropriately be in mod.rs where the integration_tests module is loaded. (Same for the other one)